setup.cfg

```
@@ -49,7 +49,7 @@ exclude =
 
 [options.extras_require]
 citations =
-    doi2bib
+    doi2bib >= 0.4.0
```

Didn't you mean

```suggestion
    doi2bib < 0.4.0
```

?